6-async/02-promise-basics/article.md

```diff
@@ -179,7 +186,7 @@ promise.then(alert); // shows "done!" after 1 second
 */!*
 ```
 
-If we're interested only in errors, then we can use `.then(null, function)` or an "alias" to it: `.catch(function)`
+If we're interested only in errors, then we can use `.catch(errorHandlingFunction)`, which is just like doing `.then(null, errorHandlingFunction)`.
```

Comment:
`...which is exactly the same`, not "just like".

Reply:
Clarified with your wording